docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_advanced_arming:
+
+***********************************************
+Advanced Arming
+***********************************************
+Vertiq modules can support an advanced arming feature, allowing the user to control the armed state of the module with throttle commands or manually 

```

Comment:
. The advanced arming feature allows users to control the module's armed state either manually or with throttle commands, and to configure specific behaviors to occur on armed state transitions.